pytest_bdd/scenario.py

```diff
@@ -330,6 +309,20 @@ def scenario(feature_name, scenario_name, encoding="utf-8", example_converters=N
     )
 
 
+def get_base_dir(caller_module):
+    default_base_dir = os.path.dirname(caller_module.__file__)
```

Comment:
this will result in the test file, not the folder containing it.

Reply:
There is `os.path.dirname` that wraps `caller_module.__file__`. It returns directory of the current test file. I think all is good here. Correct me if I am wrong.